ivy/functional/backends/numpy/experimental/linear_algebra.py

```diff
@@ -167,3 +167,37 @@ def cond(
 
 
 cond.support_native_out = False
+
+
+def solve_triangular(
+    a: np.ndarray,
+    b: np.ndarray,
+    /,
+    *,
+    lower: bool = True,
+    trans: bool = False,
```

Comment:
There is no such thing as `trans` being returned from the `Ivy API`. You are also returning `unit_diagonal` in the `Ivy API`, how are you handling it here?